test/regression/2015-08-05-invalid-fork.c

```diff
@@ -0,0 +1,24 @@
+/* Reported by @kren1 in #262 
+   The test makes sure that the string "Should be printed once" 
+   is printed a single time. 
+*/
+
+// RUN: %llvmgcc %s -emit-llvm -g -O0 -c -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out %t.bc >%t1.log
+// RUN: grep "Should be printed once"  %t1.log
```

Comment:
I say this because LLVM discouraged using `grep` in its own test suite a long time ago. Although KLEE is now an independent project `FileCheck` is a much better matching tool than `grep` is in many cases. To match only once you'd do something like

```c++
// RUN: %llvmgcc %s -emit-llvm -g -O0 -c -o %t.bc
// RUN: rm -rf %t.klee-out
// RUN: %klee --output-dir=%t.klee-out %t.bc | FileCheck %s
// ...
// CHECK: Should be printed once
// CHECK-NOT: Should be printed once
printf("Should be printed once\n");
```

I consider this to be clearer than what is in this PR because the `CHECK` directive at syntactically right next to the `printf()`.